Userland/Libraries/LibWeb/HTML/HTMLTableCellElement.h

```diff
@@ -23,6 +23,8 @@ class HTMLTableCellElement final : public HTMLElement {
     WebIDL::ExceptionOr<void> set_col_span(unsigned);
     WebIDL::ExceptionOr<void> set_row_span(unsigned);
 
+    int cell_index() const;
```

Comment:
You could use `WebIDL::Long` for the return type here.

Reply:
Done :)